src/native/fs/fs_napi.cpp

```diff
@@ -2174,13 +2174,13 @@ fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
     if (gpfs_dl_path != NULL) {
-        LOG("FS::GPFS GPFS_DL_PATH=" << gpfs_dl_path);
+        DBG1("FS::GPFS GPFS_DL_PATH=" << gpfs_dl_path);
         struct stat _stat_res;
         gpfs_lib_file_exists = stat(gpfs_dl_path, &_stat_res); //SYSCALL_OR_RETURN
         if (gpfs_lib_file_exists == -1) {
-            LOG("FS::GPFS WARN couldn't find GPFS lib file GPFS_DL_PATH=" << gpfs_dl_path);
+            DBG1("FS::GPFS WARN couldn't find GPFS lib file GPFS_DL_PATH=" << gpfs_dl_path);
         } else {
-            LOG("FS::GPFS found GPFS lib file GPFS_DL_PATH=" << gpfs_dl_path);
+            DBG1("FS::GPFS found GPFS lib file GPFS_DL_PATH=" << gpfs_dl_path);
```

Comment:
DBG1 is probably too high, DBG0 maybe better
But @guymguym said that we need to keep it LOG, maybe I miss-understood?

Reply:
I will revert back to `LOG`